config/routes.rb

```diff
@@ -3,7 +3,7 @@
   
   root "recipes#index"
 
-  resources :foods
+  resources :foods, only: [:index, :new, :create, :edit, :destroy, :update]
```

Comment:
[Optional] Since you're using all the routes for the food controller, you don't need to specify each individual action. This will keep our code clean

Reply:
Okay, thanks pointing that, will update it